### ISSUE TITLE
The accessibility Screen reader announces the previous element after focus leaves the autocomplete.   

### ISSUE THREAD

**Issue by rootlinux2** [downshift-js/downshift#862]
Hi guys, I'm facing a problem with the autocomplete, I'm using the latest version of DownShift but when I run the accessibility test in my ReactJS component I realize that the screen reader after pass for the autocomplete input the read the element before the autocomplete and then read the element after the autocomplete. I just create a code sandbox to make more clear the situation.

For the accessibility test, I'm using Mozilla Firefox and Google Chrome latest and NVDA 2019.2.

This is the sandbox with a failing example of Downshift and a successful example with HTML5

[Sandbox](https://codesandbox.io/s/nice-moon-wchzz?fontsize=14&hidenavigation=1&theme=dark)

**Comment by silviuavram** [comment]
ok not sure I'm following. example in the sandbox is good. can you please describe the scenario with test steps, expected and actual behavior?

**Comment by rootlinux2** [comment]
Test steps:
Once the page is loaded, with the keyboard you move the focus to the first element focusable in the DOM, in the sandbox example is the FIRST button, the Screen Reader (SCR for short) should announce ''first button", this is Ok, then you press the TAB key and the focus should go to the downshift autocomplete, the SCR should announce the autocomplete field, this is Ok too, then you press the TAB key again and the focus should go to the next element, the button "Second" the focus move ok but the SCR announce the First button and then announce the Second button, this is wrong, the SCR should only announce the second button.
For this example, I use a button but whatever element before the Downshift autocomplete will be announced by the SCR.
In the Sandbox I included another basic example with HTML5 autocomplete, this example works fine for accessibility matters. 

If I can help in any way, please let me know!

**Comment by silviuavram** [comment]
Voiceover reads fine.

Can you take a look at this example https://codesandbox.io/s/simple-downshift-with-getrootprops-example-24s13  and adjust the Downshift structure according to it? It should involve using `getRootProps` on the element that wraps the `input`. Then test it again with NVDA.

Thanks!

**Comment by rootlinux2** [comment]
Hi @silviuavram, I just check your sandbox, I took the license to add two buttons around the DownShift component and try again with NVDA, the ISSUE is still present, I also checked on VoiceOver, ChromeVox, You're right the Issue it is only affecting to NVDA users.
In VoiceOver if you take a deep look you will see it fail some times, for example, if you select a product like 'banana', and move the focus to another element in the page and then return to the first button, the SCR will read "First button banana", I let you a little video.  
[VoiceOver](https://www.loom.com/share/52b183e48ead4dfa830010e05fe04950)

**Comment by rootlinux2** [comment]
Hi, @silviuavram I was testing the latest version fo the Downshift library and this issue appears to be solved in this version. Thanks...
